apps/framework-cli/src/cli/commands.rs

```diff
@@ -98,6 +98,9 @@ pub enum GenerateCommand {
         /// The location of the Moose project
         #[arg(default_value = ".", short, long)]
         project_location: PathBuf,
+        /// Whether or not to generate a full fledged package or just the source files in the language of choice
+        #[arg(default_value = "false", short = 'a', long)]
+        packaged: bool,
```

Comment:
Generally I prefer short & long to start with the same letter so they're easier dx. Looks like "p" is already taken by project location. Couple more options could be `--full-package / -f` and `--complete / -c`.

Reply:
Ha thank you for the suggestion - I also prefer it that way but did not think of other alternatives. Will switch it up!